docs/src/common.md

```diff
@@ -151,36 +156,41 @@ H(q0) == J0
 
 Following the `DifferentialEquations.jl`
 [tutorial](http://docs.juliadiffeq.org/latest/tutorials/ode_example.html),
-we define an `ODEProblem` for the integration;
-`TaylorIntegration.jl` can be used via its common interface bindings with `DiffEqBase.jl`; both packages need to be loaded explicitly.
+we define an `ODEProblem` for the integration; `TaylorIntegration` can be used
+via its common interface bindings with `DiffEqBase.jl`; both packages need to
+be loaded explicitly.
 ```@example common
-tspan = (0.0, 1000.0)
+tspan = (0.0, 2000.0)
 p = [μ]
 
-using TaylorIntegration, DiffEqBase
-prob = ODEProblem(f, q0, tspan, p)
+using DiffEqBase
```

Comment:
While it doesn't do anything here, I would still load both packages for consistency to what  is written.

Reply:
Ok, re-added TaylorIntegration there